src/Exchange.ts

```diff
@@ -332,6 +276,21 @@ export class Exchange {
     return this._transaction.relayTx(signedTx)
   }
 
+  public async getLogs (
+    exchangeAddress: string,
+    filter: EventFilterOptions = {}
+  ): Promise<AnyExchangeEvent[]> {
+    const { _event, _user, _utils } = this
+    const baseUrl = `exchange/${exchangeAddress}/users/${_user.address}/events`
+    const parameterUrl = _utils.buildUrl(baseUrl, filter)
+    const rawEvents = await _utils.fetchUrl<AnyExchangeEventRaw[]>(parameterUrl)
+    const formattedEvents = await _event.setDecimalsAndFormat(rawEvents)
+    return formattedEvents.map(e => ({
+      ...e,
+      orderHash: e.orderHash.toLowerCase()
```

Comment:
is that a bug in the relay we should fix? Should it be lower case? Then please open an issue.

Reply:
ah no it was a bug but is fixed now with the schemas in the relay. removed this